tests/test_python.py

```diff
@@ -523,12 +532,45 @@ def test_clear(self):
     def test_pending_screenshots_gives_only_those_screenshots_which_are_not_collected(self):
         ScreenshotsStore.capture()
         pending_screenshots = ScreenshotsStore.pending_screenshots()
-        self.assertEqual(['foo'], pending_screenshots)
+        self.assertEqual(1, len(pending_screenshots))
+        screenshot_file = pending_screenshots[0]
         pending_screenshots = ScreenshotsStore.pending_screenshots()
-        self.assertEqual([], pending_screenshots)
+        self.assertEqual(0, len(pending_screenshots))
         ScreenshotsStore.capture()
         pending_screenshots = ScreenshotsStore.pending_screenshots()
-        self.assertEqual(['foo'], pending_screenshots)
+        self.assertEqual(1, len(pending_screenshots))
+        self.assertNotEqual(screenshot_file, pending_screenshots[0])
+
+    def test_capture_shoould_vefify_screenshot_file_for_file_based_custom_screenshot(self):
+        first_screenshot = os.path.join(
+            os.getenv("screenshots_dir"), "screenshot{0}.png".format(uuid1()))
+        second_screenshot = os.path.join(
+            os.getenv("screenshots_dir"), "screenshot{0}.png".format(uuid1()))
+
+        def returns_abs_path():
+            file = open(first_screenshot, "w")
+            file.write("screenshot data")
+            file.close()
+            return file.name
+
+        def returns_base_ath():
+            file = open(second_screenshot, "w")
+            file.write("screenshot data")
+            file.close()
+            return os.path.basename(file.name)
```

Comment:
do we really need to create actual files in the filesystem? The method can just return a dummy filename.